panels/user-accounts/um-app-permissions.c

```diff
@@ -882,6 +903,12 @@ um_app_permissions_init (UmAppPermissions *self)
   self->cancellable = g_cancellable_new ();
   self->apps = g_list_store_new (G_TYPE_APP_INFO);
 
+  self->app_info_monitor = g_app_info_monitor_get ();
+  self->app_info_changed_id = g_signal_connect (self->app_info_monitor,
+                                                "changed",
+                                                (GCallback) app_info_changed_cb,
+                                                self);
```

Comment:
Instead of storing the signal id, we've been using `g_signal_connect_object()` in Settings a lot more these days. I'd rather use it here too.

Reply:
Fixed.